dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/HttpMessageCodec.java

```diff
@@ -48,8 +50,22 @@ default Object[] decode(InputStream inputStream, Class<?>[] targetTypes) throws
 
     MediaType contentType();
 
+    default MediaType responseContentType() {
```

Comment:
Why add `responseContentType` method?

Reply:
I will do some refactor, and remove this method later :)